packages/ui/src/components/Visualization/Canvas/FormV2/fields/PasswordField.test.tsx

```diff
@@ -55,10 +55,11 @@ describe('PasswordField', () => {
       </ModelContextProvider>,
     );
 
-    const clearButton = wrapper.getByTestId(`${ROOT_PATH}__clear`);
-    act(() => {
-      fireEvent.click(clearButton);
-    });
+    const fieldActions = wrapper.getByTestId(`${ROOT_PATH}__field-actions`);
```

Comment:
It's better to wrap each action, like click, into `act` callbacks, so we exhaust all intermediate rendering steps